site/docs/v1/tech/apis/_webauthn-authenticate-start-request-body.adoc

```diff
@@ -0,0 +1,33 @@
+==== Request Body
+
+[.api]
+[field]#applicationId# [type]#[UUID]# [required]#Required#::
+The unique Id for the application the user wishes to authenticate to.
+
+[field]#credentialId# [type]#[UUID]# [optional]#Optional#::
+The unique database Id for one of a user's passkeys. If this parameter is provided, the authentication ceremony will be limited to the corresponding passkey.
+
+[field]#loginId# [type]#[String]# [optional]#Optional#::
+The email or the username of the authenticating user.
++
+This field is marked optional, but you must provide either the [field]#loginId# or the [field]#userId# to complete this API. If both fields are provided in the request body, the [field]#userId# will be utilized.
+
+[field]#state# [type]#[Object]# [optional]#Optional#::
+Additional data to be passed through this WebAuthn authentication request. After successful login, the [field]#state# field in the response will contain this data.
+
+[field]#userId# [type]#[UUID]# [optional]#Optional#::
+The unique Id of the authenticating user.
++
+This field is marked optional, but you must provide either the [field]#loginId# or the [field]#userId# to complete this API. If both fields are provided in the request body, the [field]#userId# will be utilized.
+
+[field]#workflow# [type]#[String]# [required]#Required#::
+The WebAuthn workflow for the current authentication ceremony. The selected workflow will change the passkey request options on the response based on the tenant configuration. The workflow must be enabled at the tenant level and not overridden at the application level. The possible values are:
++
+* `bootstrap` - The user is authenticating using a WebAuthn passkey after providing their email or username.
```

Comment:
Discoverable is a separate thing that we don't support yet. It could be used with the bootstrap workflow, but it doesn't have to be.

I've been debating whether bootstrap is the best term for this workflow. It can be used for any "manual" WebAuthn sign in where the user clicks the button on the normal login page, not just login on a new device. I think that's the most important distinction with the re-authentication workflow, but I'm worried that users will miss the other user case.

Reply:
Okay, is this correct:

* bootstrap: the user provides something to 'bootstrap' the authentication. could be info from the authenticator, manually typing in a username, clicking a link which has user information in it
* reauthentication: FusionAuth has placed data on the device (typically in the form of a cookie) which lets us know who the user is, so they don't have to 'bootstrap' the flow.

Is that correct?

Reply:
I was basing the terminology off of https://fido-alliance.github.io/how-to-fido/HowToFIDO.html.

I had interpreted "bootstrap" as meaning "the first time a user signs in from a device." Reading it again now, I think your interpretation may be closer to what they were going for.

I can update the docs to reflect that, but it's going to mean some changes to verbiage in-app too.